<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>editPCarDialog</class>
 <widget class="QDialog" name="editPCarDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>337</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit private car</string>
  </property>
  <widget class="QDialogButtonBox" name="EditPCarBbuttonBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>290</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>351</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <widget class="QLabel" name="label_71">
      <property name="text">
       <string>Model:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="modl4"/>
    </item>
    <item>
     <widget class="QLabel" name="label_9">
      <property name="text">
       <string>Owner Name:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owNa4"/>
    </item>
    <item>
     <widget class="QLabel" name="label_101">
      <property name="text">
       <string>Owner Surname:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owSu4"/>
    </item>
    <item>
     <widget class="QLabel" name="label_111">
      <property name="text">
       <string>Owner Civic ID:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="owID4"/>
    </item>
    <item>
     <widget class="QLabel" name="label_121">
      <property name="text">
       <string>Ensured by:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="ensu4"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>EditPCarBbuttonBox</sender>
   <signal>accepted()</signal>
   <receiver>editPCarDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>EditPCarBbuttonBox</sender>
   <signal>rejected()</signal>
   <receiver>editPCarDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
